crates/cli-support/src/js/mod.rs

```diff
@@ -1499,12 +1499,19 @@ impl<'a> Context<'a> {
                     arg = arg.slice(offset);
                     ptr = wasm.__wbindgen_realloc(ptr, size, size = offset + arg.length * 3);
                     const view = getUint8Memory().subarray(ptr + offset, ptr + size);
+                    const ret = cachedTextEncoder.encodeInto(arg, view);
```

Comment:
@alexcrichton I see you already merged it, but this change... doesn't seem right. Why is it encoding same view twice now?